packages/asc-ui/src/components/InfoBanner/InfoBannerStyle.ts

```diff
@@ -0,0 +1,93 @@
+import styled, { css, Theme } from '@datapunt/asc-core'
+import {
+  focusStyleOutline,
+  svgFill,
+  themeColor,
+  themeSpacing,
+} from '../../utils'
+import Button from '../Button'
+
+export type Level = 'normal' | 'attention' | 'error'
+
+type SharedProps = {
```

Comment:
Is there any reason we're defining this type separately?

Reply:
Yes, we use this one on ContentWrapper and CloseButtonWrapper